src/main/components/ResponseComponent.jsx

```diff
@@ -10,13 +10,10 @@ const ResponseComponent = (props) => {
 
   return (
     <div>
-      <p>{`${checkmark} ' - ' ${status}`}</p>
-      <textarea
-        cols='50'
-        rows='5'
-        value={JSON.stringify(payload)}
-        readOnly={true}
-      />
+      <p>{`${checkmark}  -  ${status}`}</p>
+      <textarea cols='50' rows='5'>
+        {payload}
```

Comment:
Oh shoot—sorry I tried to fix this merge conflict but may have messed something else up. @joepav want to take a look?